xdsl/dialects/func.py

```diff
@@ -99,6 +98,7 @@ def verify_(self) -> None:
         entry_block: Block = self.body.blocks[0]
         block_arg_types = [arg.type for arg in entry_block.args]
         if self.function_type.inputs.data != tuple(block_arg_types):
+            print(self.function_type.inputs.data)
```

Comment:
```suggestion
```
Not sure why this was added, or at least I would put it in the exception message otherwise.

Reply:
Good catch!! thanks :pray: